index.html

```diff
@@ -0,0 +1,213 @@
+            <div class="flex flex-row">
+      <!---first project-->
+        <div class=" group  bg-yellow-200 relative w-1/4  m-5  overflow-hidden">
+          <img src="./Images/2.jpg" alt="Avatar" class="object-cover w-full h-full opacity-50 transition duration-500   group-hover:opacity-100 " />
+          <div class="absolute w-full p-3 bottom-0 inset-x-0 bg-yellow-300 text-gray-800 text-lg text-center leading-5 group-hover:opacity-0">
+              <h3>KIA</h3>
+              <p>Responsive Website </p>
+          </div>
+          </div>
+           <!---second project-->
+     
+          <div class=" group  bg-blue-200 relative w-1/4  m-5  overflow-hidden">
+            <img src="./Images/1.jpg" alt="Avatar" class="object-cover w-full h-full opacity-50  transition duration-500 group-hover:opacity-100" />
+            <div class="absolute w-full p-3 bottom-0 inset-x-0 bg-blue-300 text-gray-800 text-lg text-center leading-5 group-hover:opacity-0">
+                <h3>kalleh</h3>
+                <p>Responsive Website </p>
+            </div>
+            </div>
+
+  
+     <!---third project-->
+        <div class=" group  bg-red-200 relative w-1/4  m-5  overflow-hidden">
+          <img src="./Images/3.jpg" alt="Avatar" class="object-cover w-full h-full opacity-50  transition duration-500 group-hover:opacity-100" />
+          <div class="absolute w-full p-3 bottom-0 inset-x-0 bg-red-300 text-gray-800 text-lg text-center leading-5 transition duration-500 group-hover:opacity-0">
+              <h3>Solico</h3>
+              <p>Internal Portal </p>
+          </div>
+          </div>
+          <!---forth project-->
+          <div class=" group  bg-green-200 relative w-1/4  m-5  overflow-hidden">
+            <img src="./Images/4.jpg" alt="Avatar" class="object-cover w-full h-full opacity-50 transition duration-500 group-hover:opacity-100" />
+            <div class="absolute w-full p-3 bottom-0 inset-x-0 bg-green-300 text-gray-800 text-lg text-center leading-5 group-hover:opacity-0">
+                <h3>Bonny Chow</h3>
+                <p>Document Manager System </p>
+            </div>
+            </div>
+                
+                </div>
+            
+            </div>
+      </section> 
+      <section class="container mx-12 mb-12 mt-12 ">
+    
+        <div class=" text-gray-700 mb-12 mt-12 text-center uppercase font-bold text-3xl">lets talk</div>
+          <ul class=" flex flex-row justify-center mt-4 gap-24">
+              <li class="">
+                <a
+                  href="samiraghadirzad@gmail.com"
+                  class=" text-gray-800 "
+                >
+                  <i class="fa fa-envelope  fa-2x" aria-hidden="true"></i>
+                </a>
+              </li>
+              <li>
+                <a
+                  href="https://www.linkedin.com/in/samira-ghadirzad-a5b9688a/"
+                  class="flex items-center justify-center   text-gray-800 "
```

Comment:
I'm not gonna comment on all of the formating/alignment of code issues because there are many of them, so please review the file after fixing it! Prettier will help you a lot here! :)